open_discussions/test_utils.py

```diff
@@ -50,18 +52,20 @@ def assert_not_raises():
         pytest.fail(f"An exception was not raised: {traceback.format_exc()}")
 
 
-class MockResponse:
+class MockResponse(HttpResponse):
     """
-    Mock requests.Response
+    Mocked HTTP response object that can be used as a stand-in for request.Response and
+    django.http.response.HttpResponse objects
     """
 
     def __init__(self, content, status_code):
```

Comment:
Can we add a docstring and mark `content` as `str`, so that pycharm can show a warning if bytes are passed in?